### PR TITLE
Fixes #7818: Reorganize the introduction

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/7818

### CODE REVIEW

File: 5_references/39_server_install.txt [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+Package for the database containing the logs sent by each Node and the reports
+computed by Rudder. This is a 'PostgreSQL' database using the 'PostgreSQL'
+engine of the distribution. The package has a dependency on the +postgresl+
+package, creates the database named +rudder+ and installs the inialisation
+scripts for that database in +/opt/rudder/etc/postgresql/*.sql+.
+
++rudder-cfengine-community+::
+
+Package for the CFEngine server. This server delivers to the Nodes the Applied
+Policies converted into CFEngine promises.
```
Comment (jooooooon) [inline 5_references/39_server_install.txt]: This package doesn't exist anymore. It was removed a while back, and now we just have rudder-agent everywhere (on nodes and server). You can just remove this paragraph.

**Comment by jooooooon** [comment]
Aside from that one comment, this looks great.

**Comment by amousset** [comment]
PR updated

**Comment by amousset** [comment]
Also removed the "The package is not a dependency, but its installation is recommended" for git.

File: 5_references/39_server_install.txt [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+Application server for +rudder-webapp+ and +rudder-inventory-endpoint+. Both
+packages are written in 'Scala'. At compilation time, they are converted into
++.war+ files. They need to be run in an application server. 'Jetty' is this
+application server. It depends on a compatible Java Runtime Environment. It can
+be either 'Oracle Java JRE' or 'OpenJDK 7 JRE'.
+
++rudder-techniques+::
+
+Package for the Techniques. They are installed in
++/opt/rudder/configuration-repository/techniques+. At runtime, the Techniques are
```
Comment (jooooooon) [inline 5_references/39_server_install.txt]: Just realised this path is wrong. It's opt Rudder share techniques.

File: 5_references/39_server_install.txt [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+copied into a 'git' repository in +/var/rudder+. Therefore, the package depends
```
Comment (jooooooon) [inline 5_references/39_server_install.txt]: And here, configuration repository is missing

**Comment by amousset** [comment]
PR updated

**Pull request merged**